website/blog/2022-12-01-release-0.10-blog.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Podman Desktop Release v0.10
+description: Podman Desktop v0.10 has been released!
+slug: podman-desktop-release-0.10
+authors: [deekay2310]
+tags: [podman-desktop, release, kubernetes]
+hide_table_of_contents: false
+---
+
+This blog post covers all that is new with the latest release of Podman Desktop v0.10!
+
+With Podman Desktop v0.10, you will be able to install or upgrade to the latest Podman Engine (v4.3.1). This is essential on getting started with Podman Desktop if you don’t have any other container engine configured.
+
+<!--truncate-->
+
+## Improved Kubernetes Workflow
+Podman Desktop facilitates the application development process right from containerization to Kubernetes deployment. With a focus on improving the Kubernetes workflow, we’ve introduced a set of new features.
+
+**1. Custom Kubeconfig file path**
+
+Starting v0.10, users can provide their kubeconfig path location instead of letting Podman Desktop use the default path i.e. `$HOME/.kube/config`. This can be done by going to `Settings > Preferences > Kubernetes: Kubeconfig`
+
+![custom kubeconfig file path](img/podman-desktop-release-0.10/custom-kubeconfig.png)
+
+**2. Play Kubernetes YAML from the Containers list**
+
+Development workflows can require developers to leverage existing Kubernetes YAML files. Podman Desktop now supports playing Pods or Containers from an existing Kubernetes YAML File. 
+
+This can be done in the ‘Containers’ or 'Pods' screen of the application by clicking the `Play Kubernetes YAML` button and playing the existing YAML file.
```

Comment:
it was already the case from Pod

so it should only highlight that it's only new to this page

Reply:
https://github.com/containers/podman-desktop/pull/943/commits/9ae20742cadf241c250c5faff504b3738d335f16